### PR TITLE
set timeout for kubectl commands to skip long namespace deletion request

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

e2e test stuck for 6 hours (Max VM occupation time limit) while deleting kubernetes namespace deletion. This timeout for kubectl commands will unblock unexpected long namespace deletion.

## Issue reference

n/a

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Review by tcnghia — APPROVED** [review]

**Comment by tcnghia** [comment]
This looks good but I wonder if we have some finalizers that may be waiting for control plane Pods that may have been deleted? 

In that case we probably want to delete app test pods before the controller pods.

File: tests/test-infra/clean_up.sh [hunk 1]
```diff
@@ -15,6 +15,6 @@ for app in $installed_apps; do
 done
 
 echo "Trying to delete namespace..."
-kubectl delete namespace $1
+kubectl delete namespace $1 --timeout=10m
```
Comment (mukundansundar) [inline tests/test-infra/clean_up.sh:18]: with this change will it fail with error?

**Review by yaron2 — APPROVED** [review]

**Comment by youngbupark** [comment]
> This looks good but I wonder if we have some finalizers that may be waiting for control plane Pods that may have been deleted?
> 
> In that case we probably want to delete app test pods before the controller pods.

In the happy path, e2e test framework and cleanup script deletes all pods before deleting namespace. once all pods are deleted, then namespace will be deleted.

My hypothesis is that kubectl for windows in windows VM has the bug to track the status of ns deletion. let's try this fix now.

File: tests/test-infra/clean_up.sh (hunk 1 above)
Comment (youngbupark) [reply tests/test-infra/clean_up.sh:18]: yes I tested. It is failed. The hypothesis is that kubectl for win has the bug to check the status of ns deletion request. let's see how this fix goes.

**Comment by youngbupark** [comment]
> > This looks good but I wonder if we have some finalizers that may be waiting for control plane Pods that may have been deleted?
> > In that case we probably want to delete app test pods before the controller pods.
> 
> In the happy path, e2e test framework and cleanup script deletes all pods before deleting namespace. once all pods are deleted, then namespace will be deleted.
> 
> My hypothesis is that kubectl for windows in windows VM has the bug to track the status of ns deletion. let's try this fix now.

My hypothesis is wrong. 

But, E2E test framework always ensures test app is deleted when test is done. It is stuck in terminating pods running on windows node.

youngp@DESKTOP-VCQOJ3N tmp % kubectl get pods -n dapr-tests
NAME                             READY   STATUS        RESTARTS   AGE
hellobluedapr-7d847cbdf9-pq46h   0/2     Terminating   0          7d16h

This fix will be a temporary fix to skip this node. AKS team needs to investigate this issue.

**Comment by youngbupark** [comment]
https://kubernetes.io/docs/tasks/run-application/force-delete-stateful-set-pod/#force-deletion

I can add force-delete pod command, but this could not ensure pod is deleted completely. 

AKS windows node seems unstable under our testing (frequent app deployment and deletion). I would suggest to opt out windows test from scheduled e2e test.

**Pull request merged**